hack/templates/aws_v1alpha1_ccs_accountclaim_cr.tmpl

```diff
@@ -18,14 +18,13 @@ objects:
       regions:
       - name: us-east-1
     awsCredentialSecret:
-      name: aws
-      namespace: ${NAMESPACE} 
+      namespace: ${NAMESPACE}
```

Comment:
Can we just leave the name empty?

Reply:
@fahlmant It's not empty. Line 22 has the name with the `${NAME}` variable.

Reply:
:facepalm:  Missed that